code/datums/status_effects/buffs.dm

```diff
@@ -99,6 +102,7 @@
 	alerttooltipstyle = "clockcult"
 
 /datum/status_effect/inathneqs_endowment/on_apply()
```

Comment:
real talk we need to refactor these entirely

the way status effects work is atrocious